### ISSUE TITLE
[Accessibility] Footer nests headings inside of lists in-properly

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/vets-design-system-documentation#5051]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

The Section 508 Office reports an issue with the VA.gov footer:

> **Defect 6 | Critical | Created:
> WCAG 1.3.1 Information and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.**
> Ask VA page
> The structure and relationships of a list conveyed through presentation cannot be programmatically determined. For example, when the JAWS screen reader user press “L” for list items, JAWS announces list of 5 items. Next, when the user press “I” for list items, JAWS announces items incorrectly. The list items announced are not visually part of the same group. For example, JAWS includes “Heading level 2 Call us” as part of the list items. Please see attached screenshot. (New: 9/22/2025)

The screenshot they provided:

<img width="1917" height="1027" alt="Image" src="https://github.com/user-attachments/assets/062e5a1b-abb0-45b8-b83f-55243ea65173" />

In short, the issue is with the list structure. For some reason, "Resources and support," "Contact us," "Call us," the two phone numbers, and "Visit a medical center or regional office" are all programmatically part of the same list, despite not _visually_ being in the same list.

I recommend splitting out the groups into separate lists, similar to the "Veteran programs and services" list and "More VA resources" list.

## What I expected to happen

Visual groupings of links in the footer are correctly grouped programmatically 

## Reproducing

- CSS Library version:
- Device: Macbook
- Browser: Safari

**URL(s) where this bug occurs**

- va.gov
-
-

**Steps to reproduce:**

- Inspect the source code of "Resources and support"
- Test with a screen reader: open JAWS, then press "L" until you get to the list in question

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [x] Other

### Details
The Ask VA team captured this in a ticket, more details here: https://github.com/department-of-veterans-affairs/ask-va/issues/1938

**Comment by jeana-adhoc** [comment]
@coforma-jamie - The site-wide team owns the content within the footer.  This would be something they need to address.
